test/test_split.cpp

```diff
@@ -128,6 +128,13 @@ TEST(test_split, split) {
   EXPECT_STREQ("world", tokens10.data[2]);
   ret = rcutils_string_array_fini(&tokens10);
   ASSERT_EQ(RCUTILS_RET_OK, ret);
+
+  rcutils_string_array_t tokens11 = test_split("///my//hello//world/////", '/', 3);
+  EXPECT_STREQ("my", tokens11.data[0]);
```

Comment:
@Blast545 consider to `ASSERT_EQ(tokens11.size, 3);` before indexing.

Reply:
@hidmic I followed the standard of the previous tests, should I add that to all?

Reply:
That'd indeed be better.